src/ngraph_builder.cc

```diff
@@ -22,6 +22,7 @@
 #include "ngraph/builder/autobroadcast.hpp"
 #include "ngraph/builder/numpy_transpose.hpp"
 
+#include "ngraph/op/argmax.hpp"
```

Comment:
Should probably move this above the preceding blank line, so there's one `ngraph` group and this is separate from the `tensorflow` group.